doc/upstream-status-5xx.md

```diff
@@ -0,0 +1,151 @@
+<!--
```

Comment:
I think we will have a bunch of headers for debugging in the future, whereupon we may need a doc to explain meaning of each header, i'm not sure whether this document title is suitable.

Reply:
Agree, we need a better title. Do you have any good suggestions for the title of the document.

Reply:
updated.